doc/JSON_INFO.md

```diff
@@ -411,7 +411,6 @@ Here's a quick summary of what each of the JSON files contain, broken down by fo
 |---                            |---
 | `achievements.json`           | achievements
 | `anatomy.json`                | a listing of player body parts - do not edit
-| `ascii_arts.json`             | ascii arts for item descriptions
```

Comment:
Why remove this line?

Reply:
Because there is no ascii_arts.json file

Reply:
There is a folder instead, and other similar folders dont have an entry here cos they dont have a single json file either, so I made this change